Jenkinsfile

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env groovy
 common {
   slackChannel = '#data-governance-eng'
-  upstreamProjects = 'confluentinc/rest-utils'
+  downStreamRepos = ["secret-registry", "kafka-rest", "ksql",
```

Comment:
Actually `secret-registry` should be removed and `schema-registry-plugins` should be added

Reply:
@rayokota Thats correct. Thanks for pointing that out. I believe that those dependencies just changed recently and I missed updating this PR.